src/rulesets/oas/index.json

```diff
@@ -437,9 +437,9 @@
       "severity": 0,
       "type": "validation",
       "given": [
-        "$..definitions..[?(@property !== 'properties' && (@.example !== void 0 || @['x-example'] !== void 0 || @.default !== void 0) && (@.enum || @.type || @.format || @.$ref || @.properties || @.items))]",
-        "$..parameters..[?(@property !== 'properties' && (@.example !== void 0 || @['x-example'] !== void 0 || @.default !== void 0) && (@.enum || @.type || @.format || @.$ref || @.properties || @.items))]",
-        "$..responses..[?(@property !== 'properties' && (@.example !== void 0 || @['x-example'] !== void 0 || @.default !== void 0) && (@.enum || @.type || @.format || @.$ref || @.properties || @.items))]"
+        "$..definitions..[?(@property !== 'properties' && @ && (@.example !== void 0 || @['x-example'] !== void 0 || @.default !== void 0) && (@.enum || @.type || @.format || @.$ref || @.properties || @.items))]",
```

Comment:
This was actually throwing an exception at the runtime we silently ignored.
It's related to the latest jsonpath-plus change, where they started to evaluate these script filter expressions also in case of the falsy values. Previously, if the value was `null`, it wouldn't evaluate the expression entirely, thus the error would not appear.